package.json

```diff
@@ -58,6 +58,6 @@
   ],
   "author": "Eric Dauenhauer",
   "license": "SEE LICENSE IN LICENSE",
-  "homepage": "https://github.com/ericyd/loop-supreme",
+  "homepage": "./",
```

Comment:
this seems weird to me, but without it the production build references invalid paths. Probably will need to revisit this when the project is hosted